palm/plugins/dbt/parsers/project_parser.py

```diff
@@ -0,0 +1,20 @@
+class Project:
+    """Project parser model"""
+
+    def __init__(self, **data):
+        self.name = data['name']
+        self.version = data['version']
+        self.profile = data['profile']
+        self.config_version = data['config-version']
+        self.model_paths = data.get('model-paths', 'models')
+        self.macro_paths = data.get('macro-paths', 'macros')
+        self.seed_paths = data.get('seed-paths', 'seeds')
+        self.snapshot_paths = data.get('snapshot-paths', 'snapshots')
+        self.analysis_paths = data.get('analysis-paths', 'analysis')
+        self.test_paths = data.get('test-paths', 'tests')
+        self.packages_install_path = data.get('packages-install-path', 'packages')
```

Comment:
This is only for dbt v1+. We are still supporting v0.21? Or no longer?

Reply:
Yeah... I was thinking about this, we should support it (because we need it for legacy)... but I don't have bright ideas on how we do it.

Reply:
Well in the very least we should add the install path keyword for <v1.0 in addition to this one and resolve them wherever we are using install paths from the config (which is currently no where), right?

Reply:
Added support for the legacy config!